### PR TITLE
Added spanish translation

### PR DESCRIPTION
**Pull request by juanborda**

### CODE REVIEW

File: data/po/es.po [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+#: minigalaxy/constants.py:18
+msgid "Japanese"
+msgstr "Japones"
+
+#: minigalaxy/ui/preferences.py:40
+msgid ""
+"Keep installers after downloading a game.\n"
+"Installers are stored in: {}"
+msgstr ""
+"Guardar los instaladores luego de descargar un juego. Los instaladores se "
```
Comment (sharkwouter) [inline data/po/es.po:146]: \n is missing at the end of this sentence

**Comment by juanborda** [comment]
Hi @sharkwouter ! Nice catch, should i also update readme.md with Spanish as an available language? 

Cheers!

**Comment by sharkwouter** [comment]
Well, I tend to update the README when there is a release. If you could update the changelog that would save me some work, though :)

**Comment by sharkwouter** [comment]
Nevermind the changelog, I guess. I'll fix this. What would help is if you'd add your translation to setup.py. It's not a requirement for me to merge it, though, just fix the one comment I made and I can merge it as well.

**Comment by juanborda** [comment]
Sorry @sharkwouter ! Its been a rough couple of days, ill make sure to push this today!

**Pull request merged**